### PR TITLE
Update keyboard shortcuts to meet new style guidelines

### LINKED ISSUE #13895: Update keyboard shortcuts to meet the new GitHub content style guide
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax

### What part(s) of the article would you like to see updated?

Keyboard shortcuts in this document are outdated. Refer to [Keyboard Shortcuts](https://github.com/github/docs/blob/main/contributing/content-style-guide.md#keyboard-shortcuts) section in the content style guide on how to correctly present keyboard shortcuts in GitHub docs.

### Additional information

For example

<kbd>command</kbd><kbd>k</kbd> should be <kbd>Command</kbd>+<kbd>K</kbd>
<kbd>command</kbd><kbd>shift</kbd><kbd>7</kbd> should be <kbd>Command</kbd>+<kbd>Shift</kbd>+<kbd>7</kbd>

**This issue is for the Campus Experts + GitHub Docs program and is not available to be assigned.**

### PR DESCRIPTION
**Pull request by AnilSeervi**
### Why:

Closes #13895 

### Check off the following:

- [ ] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

File: content/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ For more information, see "[Creating and highlighting code blocks](/articles/cre
 
 ## Links
 
-You can create an inline link by wrapping link text in brackets `[ ]`, and then wrapping the URL in parentheses `( )`. {% ifversion fpt or ghae or ghes > 3.1 or ghec %}You can also use the keyboard shortcut `command + k` to create a link.{% endif %}{% ifversion fpt or ghae-issue-5434 or ghes > 3.3 or ghec %} When you have text selected, you can paste a URL from your clipboard to automatically create a link from the selection.{% endif %}
+You can create an inline link by wrapping link text in brackets `[ ]`, and then wrapping the URL in parentheses `( )`. {% ifversion fpt or ghae or ghes > 3.1 or ghec %}You can also use the keyboard shortcut <kbd>Command</kbd>+<kbd>K</kbd> to create a link.{% endif %}{% ifversion fpt or ghae-issue-5434 or ghes > 3.3 or ghec %} When you have text selected, you can paste a URL from your clipboard to automatically create a link from the selection.{% endif %}
```
Comment (AnilSeervi) [inline content/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax.md:85]: Should to brackets `[  ]` and parentheses `(  )` also be within `<kbd>`

**Comment by ramyaparimi** [comment]
@AnilSeervi 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: content/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax.md [hunk 2]
```diff
@@ -29,15 +29,15 @@ You can indicate emphasis with bold, italic, or strikethrough text in comment fi
 
 | Style | Syntax | Keyboard shortcut | Example | Output |
 | --- | --- | --- | --- | --- |
-| Bold | `** **` or `__ __`| command/control + b | `**This is bold text**` | **This is bold text** |
-| Italic | `* *` or `_ _`     | command/control + i | `*This text is italicized*` | *This text is italicized* |
+| Bold | `** **` or `__ __`| <kbd>Command</kbd>+<kbd>B</kbd> (Mac) or <kbd>Control</kbd>+<kbd>B</kbd> (Windows/Linux) | `**This is bold text**` | **This is bold text** |
+| Italic | `* *` or `_ _`     | <kbd>Command</kbd>+<kbd>I</kbd> (Mac) or <kbd>Control</kbd>+<kbd>I</kbd> (Windows/Linux) | `*This text is italicized*` | *This text is italicized* |
```
Comment (ramyaparimi) [inline content/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax.md:33]: @AnilSeervi Please use <kbd>Ctrl</kbd>+<kbd>I</kbd>  for Windows/Linux

Comment (AnilSeervi) [reply content/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax.md]: @ramyaparimi done ✨

**Review by ramyaparimi — DISMISSED** [review]
@AnilSeervi 
Thanks so much for opening a PR with this fix! I'll get this merged once tests are passing 🍏

**Review by ceoaungmyo — COMMENTED** [review]
https://www.facebook.com/ceoaungmyo

File: content/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax.md (hunk 1 above)
Comment (Sheikhmohd) [reply content/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax.md:85]: Hu

Comment (Sheikhmohd) [reply content/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax.md:85]:
```suggestion
You can create an inline link by wrapping link text in brackets `[ ]`, and then wrapping the URL in parentheses `( )`. {% ifversion fpt or ghae or ghes > 3.1 or ghec %}You can also use the keyboard shortcut <kbd>Command</kbd>+<kbd>K</kbd> to create a link.{% endif %}{% ifversion fpt or ghae-issue-5434 or ghes > 3.3 or ghec %} When you have text selected, you can paste a URL from your clipboard to automatically create a link from the selection.{% endif %}
```

**Review by ramyaparimi — APPROVED** [review]

**Pull request merged**